src/snowflake/telemetry/_internal/exporter/otlp/proto/logs/__init__.py

```diff
@@ -136,71 +119,6 @@ def _translate(self, record: logging.LogRecord) -> _logs.LogRecord:
         return otel_record
 
 
-class _SnowflakeTelemetryLogEmitter(_logs.LogEmitter):
```

Comment:
@sfc-gh-bdrutu `LogEmitter` has been removed from opentelemetry-python in 1.23.0. Do we need to reimplement the custom instrumentation scope logic here on top of 1.23.0?

Reply:
Do we have tests to tell us? :)

Reply:
Good point, I'll write some asserts for this aspect of the log data protos in our log exporter test.

Reply:
Let's have the tests in a separate PR, so we can confirm they pass before and after the upgrade.

Reply:
This seems to be a known issue with opentelemetry-python: https://github.com/open-telemetry/opentelemetry-python/issues/2485

I'll investigate the best class to override in opentelemetry-python to get the same behavior we had in snowflake-telemetry-python v0.3.0 and prior. The latest 1.23.0 release of opentelemetry-python eliminates the LogEmitter and LogEmitterProvider classes that we were previously overriding.

After that, I'll work on creating a patch for issue 2485 for public review in opentelemetry-python so that long term we don't have to maintain this fix ourselves.

Reply:
I've applied Alexey's patch on top of 1.23.0.

- LogEmitter -> Logger
- LogEmitterProvider -> LoggerProvider